### PR TITLE
feat: Now button on create disruption page

### PR DESCRIPTION
**Pull request by alainajaffer**
To test:

Go to the create disruption page and then scroll to publication date and time
As per wireframes should be a 'Now' button click on it to see the current date and time

### CODE REVIEW

File: site/pages/create-consequence-network/__snapshots__/create-consequence-network.test.tsx.snap [hunk 1]
```diff
@@ -277,17 +277,21 @@ exports[`pages > CreateConsequenceNetwork > should render correctly with inputs
...
-                  className="govuk-input govuk-date-input__input govuk-input--width-4"
-                  defaultValue="yes"
-                  disabled={false}
-                  id="disruption-delay-input"
-                  name="disruptionDelay"
-                  onBlur={[Function]}
-                  placeholder=""
-                  type="text"
-                />
+                <div
```
Comment (alvak1) [inline site/pages/create-consequence-network/__snapshots__/create-consequence-network.test.tsx.snap:280]: why is this div added?

Comment (alainajaffer) [reply site/pages/create-consequence-network/__snapshots__/create-consequence-network.test.tsx.snap:280]: for css when the now button is show notice how on the time selector a flex and gap is added

File: site/components/form/TimeSelector.tsx [hunk 2]
```diff
@@ -53,20 +71,29 @@ const TimeSelector = <T extends object>({
...
+                            disabled={disabled}
+                            placeholder={disabled ? "N/A" : placeholderValue}
+                            aria-describedby={hint ? `${inputId}-hint` : ""}
+                            onBlur={(e) =>
+                                handleBlur(e.target.value, inputName, stateUpdater, setErrors, schema, disabled)
+                            }
+                        />
+                    </FormElementWrapper>
+                    {showNowButton ? (
+                        <button className="mt-3 govuk-link" data-module="govuk-button" onClick={showNowButton}>
```
[the remark this answers is not in the document]
Comment (alainajaffer) [reply site/components/form/TimeSelector.tsx:92]: this is gov component css will need to discuss if i can change that with chris

[the remark this answers is not in the document]
Comment (alainajaffer) [reply site/components/form/TimeSelector.tsx]: done

**Review by alvak1 — APPROVED** [review]
Approved

**Pull request merged**